### PR TITLE
Initial FAQ, with hopefully more contributions in the future.

### PR DESCRIPTION
**Pull request by stair-aws**
*Issue #, if available:*
N/A

*Description of changes:*
Initial FAQ, with hopefully more contributions in the future.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by stair-aws** [comment]
Default IntelliJ Markdown auto-formatting is, subjectively, counter-productive and, objectively, unnecessarily verbose.

Fortunately, there's a way to disable some unhelpful IntelliJ edits: https://blog.jetbrains.com/webstorm/2023/01/webstorm-2023-1-eap-1/#new_smart_keys_settings_page_for_markdown (i.e., disable "Reformat table")

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Frequently Asked Questions (FAQ)
```
Comment (brendan-p-lynch) [inline docs/FAQ.md:1]: This is a great idea. Can we add this FAQ as a link to the README for increased visibility?

**Review by brendan-p-lynch — APPROVED** [review]
Overall I think it is a great idea. I want to make sure that if we are adding to this document that it is visible enough for people to find, and that there isn't a better place for an FAQ (like an amazon owned webpage) but personally I think keeping it close to the code is the best

File: docs/FAQ.md (hunk 1 above)
Comment (stair-aws) [reply docs/FAQ.md:1]: SGTM. Will follow-up w/ a separate PR soas not to unnecessarily delay merging this incremental value.

**Pull request merged**